### DISCUSSION TITLE
Aerofoil Mode - print skin first.

### DISCUSSION THREAD

**Discussion by VinceHogg** [OrcaSlicer/OrcaSlicer#7736] [Ideas]
May I suggest an 'aerofoil mode' that prints the outside skin first. Its NOT the same as the inside/outside setting which only works on 2 or more walls. In 'direct design' wing method the skin is usually a single wall and the inside  ribs are separated from the skin by about 0.01mm. This makes the skin print in an uninterrupted loop but the ribs are so close they do join up to the skin. Problem - This system works well BUT if the ribs are printed first the skin extrusion can be pushed out when passing them. If the skin was printed before the ribs this probably wouldn't happen. See the pic. Some of the ribs are printed before the skin. I have not found a way to stop this and don't believe there is one so this is a feature request.
![orca](https://github.com/user-attachments/assets/fdf51671-12fc-424e-94be-8ba62129bd01)

**Comment by DARKFiB3R** [comment]
What happens if you split the external wall and the internal structure into 2 separate objects?
